File: cpp/1722A.cpp
#include <algorithm>
#include <cmath>
#include <cstdio>
#include <iostream>
#include <map>
#include <queue>
#include <set>
#include <stack>
#include <string>
#include <unordered_map>
#include <vector>

using namespace std;

int main() {
    ios::sync_with_stdio(false);
    int t, n;
    cin >> t;
    while (t--) {
        cin >> n;
        string name;
        cin >> name;
        if (n != 5) {
            cout << "NO" << endl;
            continue;
        }
        sort(name.begin(), name.end());
        if (name == "Timru") {
            cout << "YES" << endl;
        } else {
            cout << "NO" << endl;
        }
    }
    return 0;
}

File: goSolution/1722A.go
package goSolution

import (
	"fmt"
	"sort"
)

func Solve_1722A() {
	var t, n int
	var name string
	fmt.Scan(&t)
	for t > 0 {
		fmt.Scan(&n)
		fmt.Scan(&name)
		if n != 5 {
			fmt.Println("NO")
		} else {
			na := []byte(name)
			sort.Slice(na, func(i, j int) bool { return na[i] < na[j] })
			if string(na) == "Timru" {
				fmt.Println("YES")
			} else {
				fmt.Println("NO")
			}
		}
		t--
	}
}
